src/compile/selection/point.ts

```diff
@@ -52,21 +92,38 @@ const point: SelectionCompiler<'point'> = {
       update += `fields: ${fieldsSg}, values: [${values}]`;
     }
 
-    const events: Stream[] = selCmpt.events;
-    return signals.concat([
-      {
-        name: name + TUPLE,
-        on: events
-          ? [
-              {
-                events,
-                update: `${test} ? {${update}} : null`,
-                force: true
-              }
-            ]
-          : []
-      }
-    ]);
+    if (isTimerSelection(selCmpt)) {
+      // timer event: selection is for animation
+      // TODO(jzong) scale name currently hardcoded to 'time'
```

Comment:
Status of this TODO?

Reply:
resolved in https://github.com/vega/vega-lite/pull/8921/commits/41ae4b530c4ebf22c9d3e10d4557d8d248def96d